sentry-ruby/lib/sentry/transport.rb

```diff
@@ -173,6 +168,19 @@ def fetch_pending_client_report
 
       [item_header, item_payload]
     end
+
+    def process_rate_limits(envelope)
+      envelope.items.reject! do |item|
+        if is_rate_limited?(item.type)
+          log_info("[Transport] Envelope item [#{item.type}] not sent: rate limiting")
```

Comment:
Not related to this PR: I'm thinking downgrading most of the info logs to `debug` level. Because from my experience, we rarely need to know the sent events, not even in debugging. Wdyt?

Reply:
hmm let's do this in a separate PR, I'll need to change a lot of specs otherwise :)

Reply:
Yeah I didn't mean we should change it here. And we should also change some other places instead of just here..